packages/react-core/src/components/Nav/examples/NavLinkText.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Nav, NavExpandable, NavItem, NavList } from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+import UserIcon from '@patternfly/react-icons/dist/esm/icons/user-icon';
+
+export const NavLinkText: React.FunctionComponent = () => {
+  const [activeGroup, setActiveGroup] = React.useState('nav-expand3rd-group-1');
+  const [activeItem, setActiveItem] = React.useState('nav-expand3rd-group-1_item-1');
+
+  const onSelect = (
+    _event: React.FormEvent<HTMLInputElement>,
+    result: { itemId: number | string; groupId: number | string }
+  ) => {
+    setActiveGroup(result.groupId as string);
+    setActiveItem(result.itemId as string);
+  };
+
+  const onToggle = (
+    _event: React.MouseEvent<HTMLButtonElement>,
+    result: { groupId: number | string; isExpanded: boolean }
+  ) => {
+    // eslint-disable-next-line no-console
+    console.log(`Group ${result.groupId} expanded? ${result.isExpanded}`);
+  };
+
+  return (
+    <Nav onSelect={onSelect} onToggle={onToggle} aria-label="Nav link text">
+      <NavList>
+        <NavItem
+          preventDefault
+          id="link-text-link-1"
+          to="#link-text-link-1"
+          itemId="link-text-1-item-1"
+          isActive={activeItem === 'link-text-1-item-1'}
+          hasNavLinkWrapper
+        >
+          Link 1
+          <ArrowRightIcon />
+        </NavItem>
+        <NavExpandable
+          title={
+            <>
+              Link 2<small>(small text)</small>
+            </>
+          }
+          groupId="link-text-group-1"
+          isActive={activeGroup === 'link-text-group-1'}
+          isExpanded
+        >
+          <NavItem
+            preventDefault
+            id="link-text-link-2"
+            to="#link-text-link-2"
+            groupId="link-text-group-1"
+            itemId="link-text-group-1_item-1"
+            isActive={activeItem === 'link-text-group-1_item-1'}
+          >
+            <UserIcon />
+            Subnav link 1
```

Comment:
Did you mean to add `hasNavLinkWrapper` to this and the one below? With adding the space to the 2 items I left comments about, and adding `hasNavLinkWrapper` here and the next item, it would look like this:

<img width="838" alt="Screenshot 2023-10-17 at 1 59 55 PM" src="https://github.com/patternfly/patternfly-react/assets/35148959/7bf51dbe-bec8-45b1-8261-8362826c0bf4">

Here's what it looks like now:

<img width="839" alt="Screenshot 2023-10-17 at 2 00 15 PM" src="https://github.com/patternfly/patternfly-react/assets/35148959/72cda167-9666-4c1b-b7bf-70dfb15dab13">